internal/controllers/service/tests/service-create-full/00-assert.yaml

```diff
@@ -5,9 +5,10 @@ metadata:
   name: service-create-full
 status:
   resource:
-    name: service-create-full-override
-    description: Service from "create full" test
-    # TODO(scaffolding): Add all fields the resource supports
+    name: service-create-full
```

Comment:
Let's still make sure we can override the service name (i.e. the value from resource.name takes precedence over metadata.name)